pyirf/tests/test_integration.py

```diff
@@ -0,0 +1,50 @@
+import glob
```

Comment:
I'd like it better if we create a subdirectory `integration_tests` and this would be called `test_eventdisplay_example`.

Reply:
should we really need more than one integration test?
I thought we decided that there is no need for other example scripts.

perhaps we could have max 2 integration tests: one for point-like and one for full-enclosure?

Reply:
in that case I think 1 test_integration.py would suffice with such 2 tests inside

Reply:
Yes, in the end we should have integration tests for full enclosure and point like irfs and sensitivity.

But we can come up with more integration tests, e.g. if we implement different cut optimization schemes

Reply:
Wouldn't this go the current API-based approach we decided?
We shouldn't test any specific schemes, only check that given one example scheme (which we already have) we are able to produce correctly the required output.
Otherwise, we should have to take into account all the possible schemes, which is impossible.